inspirehep/records/indexer/cli.py

```diff
@@ -192,6 +231,6 @@ def _finished_tasks_count():
         fg=color,
     )
     if failures:
-        logger.error(f"Got {len(failures)} during the reindexing process:")
+        LOGGER.error("Got '%s' failures during the reindexing process:", len(failures))
```

Comment:
same as above. Actually, being this a CLI, isn't it better to print to stdout?

Reply:
Actually we are writing this to stdout in cli
as config says:

> `
> "handlers": {
>             "stdout_handler":
> ...
> `